lineman/app/modules/navbar/navbar_search_controller.coffee

```diff
@@ -15,9 +15,9 @@ angular.module('loomioApp').controller 'NavbarSearchController', ($scope, UserAu
   $scope.getSearchResults = (query) ->
     if query?
       $scope.searching = true
-      Records.search_results.fetchByFragment($scope.query).then (response) ->
+      Records.searchResults.fetchByFragment($scope.query).then (response) ->
         $scope.searchResults = _.map response.search_results, (result) ->
-          Records.search_results.initialize result
+          Records.searchResults.initialize result
```

Comment:
camelizing means we don't change our naming system for just one place for no reason.